views/reducers/CharacterCamera.js

```diff
@@ -0,0 +1,46 @@
+import ACTIONS from '../actions';
+import {
+    nextPosition
+} from './Movement.js';
+import {
+    CharacterReducer,
+    CharacterInitialState
+} from './Character';
+import {
+    CameraReducer,
+    CameraInitialState
+} from './Camera';
+const Three = require('three');
+
+const InitialState = {
+    ...CameraInitialState,
```

Comment:
Why you are duplicating the `CameraInitialState` inside the `Character Reducer`?